api/install_test.go

```diff
@@ -84,8 +87,16 @@ func TestAPI_FunctionInstall_HandlesErrors(t *testing.T) {
 		err = api.Install(ctx)
 		require.NoError(t, err)
 
-		require.Equal(t, http.StatusRequestTimeout, rec.Result().StatusCode)
+		var res response.InstallFunction;
```

Comment:
```suggestion
		var res response.InstallFunction
```

Reply:
I only now noticed that in `install.go` we do:

```go
	// return inner code as body
	return ctx.JSON(200, map[string]interface{}{
		"code": strconv.Itoa(status),
	})
```

For consistency, I'd do:

```go
import (
	"github.com/blocklessnetworking/b7s/models/api/response"
)

// ...

res := response.InstallFunction{
	Code: strconv.Itoa(status),
}

return ctx.JSON(http.StatusOK, res)
```

Reply:
Love it, thanks for learning me some golang :)